packages/cactus-plugin-ledger-connector-iroha/src/test/typescript/integration/run-transaction-endpoint-v1.test.ts

```diff
@@ -81,6 +83,8 @@ test(testCase, async (t: Test) => {
     postgresHost: postgresHost,
     postgresPort: postgresPort,
     logLevel: logLevel,
+    tlsCert: "/opt/iroha_data/torii_tls/server",
+    tlsKey: "/opt/iroha_data/torii_tls/server",
   });
```

Comment:
Adding certificates does not make Iroha server to use them, You need to set up those patches and port to config file:
```
"torii_tls_params": {
    "port": 55552,
    "key_pair_path": "/path/to/server"
}
```